drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/assessment.rb

```diff
@@ -13,14 +13,30 @@ class Assessment < GrdaWarehouse::Hud::Base
     self.table_name = 'hmis_2020_assessments'
 
     has_one :destination_record, **hud_assoc(:AssessmentID, 'Assessment')
+
     def self.hmis_validations
       {
+        EnrollmentID: [
+          class: HmisCsvValidation::NonBlank,
+        ],
+        PersonalID: [
+          class: HmisCsvValidation::NonBlank,
+        ],
         AssessmentDate: [
           class: HmisCsvValidation::NonBlank,
         ],
         AssessmentLocation: [
           class: HmisCsvValidation::NonBlank,
         ],
+        AssessmentType: [
+          {
+            class: HmisCsvValidation::NonBlankValidation,
```

Comment:
I'm curious how you decided when blank is an error, and when it was a warning.

Reply:
If it blows up because we have a requirement at the DB level, it had to be a NonBlank, if it it's required, but we didn't implement that on the DB, then NonBlankValidation.